.circleci/config.yml

```
@@ -32,56 +43,71 @@ jobs:
             - _build
             - deps
 
-  build_elixir_1_13_otp_24:
-    docker:
-      - image: hexpm/elixir:1.13.1-erlang-24.2-alpine-3.15.0
-    steps:
-      - checkout
-      - <<: *install_hex_rebar
-      - restore_cache:
-          keys:
-            - v1-mix-cache-{{ checksum "mix.lock" }}
-      - run: mix deps.get
-      - run: mix compile
-
-  build_elixir_1_12_otp_24:
-    docker:
-      - image: hexpm/elixir:1.12.1-erlang-24.0.2-alpine-3.13.3
-    steps:
+     steps:
       - checkout
+      - <<: *install_system_deps
       - <<: *install_hex_rebar
       - restore_cache:
           keys:
-            - v1-mix-cache-{{ checksum "mix.lock" }}
+            - v2-mix-cache-{{ checksum "mix.lock" }}
+      - run: mix hex.organization auth smartrent --key $SMARTRENT_HEX_KEY
       - run: mix deps.get
-      - run: mix compile
+      - run: mix deps.unlock --check-unused
+      - run: mix compile --warnings-as-errors
+      - run: mix format --check-formatted
+      - run: mix credo --all
+      - run: mix test
+      - store_test_results:
+          path: test-junit-report.xml
+      - run: mix docs
+      - run: mix hex.build
+      - run: mix dialyzer
+      - save_cache:
+          key: v2-mix-cache-{{ checksum "mix.lock" }}
+          paths:
+            - _build
+            - deps
 
-  build_elixir_1_11_otp_23:
+  build_elixir_1_14_otp_25:
     docker:
-      - image: hexpm/elixir:1.11.2-erlang-23.1.2-alpine-3.12.1
+      - image: hexpm/elixir:1.14.3-erlang-25.2.3-alpine-3.18.0
+    <<: *defaults
     steps:
       - checkout
       - <<: *install_hex_rebar
+      - <<: *install_system_deps
       - run: mix deps.get
       - run: mix compile
       - run: mix test
 
-  build_elixir_1_10_otp_23:
+  build_elixir_1_13_otp_25:
     docker:
-      - image: hexpm/elixir:1.10.4-erlang-23.1.2-alpine-3.12.1
+      - image: hexpm/elixir:1.13.4-erlang-25.3.2-alpine-3.18.0
+    <<: *defaults
     steps:
       - checkout
       - <<: *install_hex_rebar
+      - <<: *install_system_deps
       - run: mix deps.get
       - run: mix compile
       - run: mix test
 
+  build_elixir_1_12_otp_24:
+      docker:
+        - image: hexpm/elixir:1.12.1-erlang-24.0.2-alpine-3.13.3
+        <<: *defaults
+      steps:
+        - checkout
+        - <<: *install_hex_rebar
+        - <<: *install_system_deps
+        - run: mix deps.get
+        - run: mix compile
+        - run: mix test
```

```suggestion
    docker:
      - image: hexpm/elixir:1.12.1-erlang-24.0.2-alpine-3.13.3
    <<: *defaults
    steps:
      - checkout
      - <<: *install_hex_rebar
      - <<: *install_system_deps
      - run: mix deps.get
      - run: mix compile
      - run: mix test
```

These indents are all off which is breaking the CircleCI ability to parse the config

---

fixed